### PR TITLE
feat: Add loadBalancerIP option to service

### LINKED ISSUE #13: Add support for Service -> loadBalancerIP
loadBalancerIP is described here: https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer

I just wanted to use it but noticed there is no option in the chart right now.

### PR DESCRIPTION
**Pull request by williambrode**
Fixes #13

### CODE REVIEW

**Comment by Sajfer** [comment]
@jdolitsky Sorry to ping you, but can this PR be merged?

**Review by mkilchhofer — APPROVED** [review]

File: src/chartmuseum/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Host your own Helm Chart Repository
 name: chartmuseum
-version: 3.1.0
+version: 3.1.1
```
Comment (mkilchhofer) [inline src/chartmuseum/Chart.yaml:4]: Needs a version bump. 3.1.1 was released several days ago

**Review by mkilchhofer — CHANGES_REQUESTED** [review]

File: src/chartmuseum/Chart.yaml (hunk 1 above)
Comment (jdolitsky) [reply src/chartmuseum/Chart.yaml:4]: yes, +1 then OK to merge. Should be `3.2.0`

**Comment by Sajfer** [comment]
@williambrode Any chance you can update the PR?

**Comment by Sajfer** [comment]
Can we get a release with this change?

**Comment by jdolitsky** [comment]
@Sajfer - looks like github actions never ran yesterday. I made another commit to trigger it and release should be available now:
```
$ helm repo add chartmuseum https://chartmuseum.github.io/charts --force-update 
"chartmuseum" has been added to your repositories

$ helm search repo chartmuseum/
NAME                   	CHART VERSION	APP VERSION	DESCRIPTION                        
chartmuseum/chartmuseum	3.2.0        	0.13.1     	Host your own Helm Chart Repository
```

**Pull request merged**